README.md

```diff
@@ -1 +1,69 @@
 # Professional README Generator
+
+![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)
+<br>
+
+## Description
+
+A command-line application which accepts user's input and based on it dynamically generates a professional README file for a new project using the inquirer package. The newly generated README will be found in the assets/js folder.
+<br>
+<br>
+
+## Motivation
+
+The purpose of this app is to practice working with node.js, npm, inquirer, fs, and more.
+<br>
+<br>
+Link to the deployed app: [here](https://jitasek.github.io/generate-readme/)
+<br>
+Link to the video guide: [here](https://drive.google.com/file/d/1E0dAvRmjQUrE5KG02HClb8i7FoRieMIm/view)
+
+(If the video quaility on the link above is not sufficient for you, please download the video guide file from the repo and play it locally.)
+<br>
```

Comment:
You might not need the br tags here, a regular space created using the enter key should be sufficient :-)

Reply:
Thank you, Dom!